### PR TITLE
SpringBoot liquibase configuration

### PR DESCRIPTION
**Pull request by gmackay**
Spring Boot runs the liquibase changelog on startup.

### CODE REVIEW

File: vcm/pom.xml [hunk 1]
```diff
@@ -28,6 +28,11 @@
 			<version>${postgres.version}</version>
 		</dependency>
 
+        <dependency>
+            <groupId>org.liquibase</groupId>
+            <artifactId>liquibase-core</artifactId>
```
Comment (seamussingh) [inline vcm/pom.xml]: The version number of the dependency needs to be set in the properties section> see postgres above.

File: vcm/pom.xml [hunk 2]
```diff
@@ -9,16 +9,28 @@
...
+
+        <admin-url>jdbc:postgresql://db:5432/</admin-url>
+        <admin-user>postgres</admin-user>
+        <admin-password>qwerty</admin-password>
+
+        <database>cshr</database>
+        <database-url>jdbc:postgresql://db:5432/cshr</database-url>
+        <database-user>postgres</database-user>
+        <database-password>qwerty</database-password>
+        <postgres.version>9.4-1206-jdbc42</postgres.version>
```
Comment (seamussingh) [inline vcm/pom.xml]: Is this the correct version to be used or the one in line 68 which appears to be a lower version.

File: vcm/pom.xml [hunk 3]
```diff
@@ -36,4 +53,29 @@
...
+            <plugin>
+                <groupId>org.liquibase</groupId>
+                <artifactId>liquibase-maven-plugin</artifactId>
+                <version>3.5.3</version>
+
+                <dependencies>
+                    <dependency>
+                        <groupId>postgresql</groupId>
+                        <artifactId>postgresql</artifactId>
+                        <version>9.1-901-1.jdbc4</version>
```
Comment (seamussingh) [inline vcm/pom.xml]: Use the ${} injection of value whatever the value resolves to be - see previous comment in properties section about which version should be in play.

File: vcm/pom.xml (hunk 2 above)
Comment (reachsaurabh) [reply vcm/pom.xml]: Lets revisit this as a team some time , and correct potential issues  , merging this on the promise we do this sooner.Cheers

**Pull request merged**